server/src/test/java/org/candlepin/model/EntitlementCuratorTest.java

```diff
@@ -16,23 +16,26 @@
 
 import static org.junit.Assert.*;
 
+import java.sql.SQLIntegrityConstraintViolationException;
```

Comment:
Import order is out of whack here.  Import `project_conf/eclipse/candlepin.importorder` into Preferences -> Java -> Code Style -> Organize Imports